app/views/ExposureHistory/HasBeenExposedDetail.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import dayjs from 'dayjs';
+
+import { Typography } from '../../components/Typography';
+
+import {
+  Typography as TypographyStyles,
+  Outlines,
+  Colors,
+  Spacing,
+} from '../../styles';
+
+type Posix = number;
+
+interface Exposure {
```

Comment:
this should get pulled up in a future commit, leaving here until its clear what the data model wants to be.